phpunit.xml.dist

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0"?>
-<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="./vendor/autoload.php" colors="true"
```

Comment:
Adding a schema is correct! Please use the installed version:

```xml
<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
```

https://github.com/mezzio/mezzio/blob/1925171d5647821853dabbf2c5e5975b68ae9cc6/phpunit.xml.dist#L2-L5

Reply:
@gsteel
Sorry because of the misunderstanding. Thanks for your help! 👍

Reply:
No worries @froschdesign - it's good to keep things tight ;)

I've been doing quite a bit of donkey work on the GHA and Psalm stuff - Happy to do some more in other repos to help out as long as the help is wanted?